src/main.ts

```diff
@@ -0,0 +1,11 @@
+import * as globby from 'globby';
+
+async function findFiles(path: string): Promise<string[]> {
```

Comment:
Please use ES module syntax instead of `module.exports`

```ts
async function findFiles(path: string): Promise<string[]> {
  // ...
}
```

Reply:
Add a comment for this function

Reply:
Done